src/vm/memory.c

```diff
@@ -101,7 +101,7 @@ static void blackenObject(DictuVM *vm, Obj *object) {
             ObjClass *klass = (ObjClass *) object;
             grayObject(vm, (Obj *) klass->name);
             grayObject(vm, (Obj *) klass->superclass);
-            grayObject(vm, (Obj *) klass->annotations);
+            grayObject(vm, (Obj *) klass->classAnnotations);
```

Comment:
Doing my own internal testing, this and `methodAnnotations` shouldn't need to be grey'd since they'd be in the functions constant table